pkg/oci/utils.go

```diff
@@ -255,6 +255,62 @@ func containerDeviceInfos(spec CompatOCISpec) ([]vc.DeviceInfo, error) {
 	return devices, nil
 }
 
+func containerCapabilities(s CompatOCISpec) (vc.LinuxCapabilities, error) {
+	capabilities := s.Process.Capabilities
+	var c vc.LinuxCapabilities
+
+	// In spec v1.0.0-rc4, capabilities was a list of strings. This was changed
+	// to an object with v1.0.0-rc5.
+	// Check for the interface type to support both the versions.
+	switch caps := capabilities.(type) {
+	case map[string]interface{}:
+		for key, value := range caps {
+			switch val := value.(type) {
+			case []interface{}:
+				var list []string
+
+				for _, str := range val {
+					list = append(list, str.(string))
+				}
+
+				switch key {
+				case "bounding":
+					c.Bounding = list
+				case "effective":
+					c.Effective = list
+				case "inheritable":
+					c.Inheritable = list
+				case "ambient":
+					c.Ambient = list
+				case "permitted":
+					c.Permitted = list
+				}
+
+			default:
+				return c, fmt.Errorf("Unexpected format for capabilities: %v", caps)
+			}
+		}
+	case []interface{}:
```

Comment:
I would expect this to be:
```go
case []string:
```
This way, there is no need for another cast into a string later.